riscv-router.md

```diff
@@ -12,6 +12,8 @@ Selling a low-cost, low-power, small (2-6 port) gigabit router that's easy to ha
 - operations / legal
 - [licensing](./licensing.md)
 
+Obviously it should spit out UART somewhere so we can plug Vgaterm into it.
```

Comment:
:laughing:  Good news: a serial console is absolutely A Thing that we're gonna want to figure out for this device as a separate access path if the network bits get borked.